ThePhasesOfCraftship/2_best_practice_first/testingBasics/exercises/1_Red_Green_Refactor/1_1_Fizzbuzz/src/index.spec.ts

```diff
@@ -1,5 +1,59 @@
+import { fizzbuzz } from "./fizzbuzz";
 
 describe("fizzbuzz", () => {
+  it("should return string", () => {
+    expect(typeof fizzbuzz(5)).toBe("string");
+  });
```

Comment:
I think you can remove this because it tests implementation details instead of the core business logic :)

Reply:
Keep it because of Khalil 1st video, but agree it's redundant